.github/workflows/docs-ci.yml

```diff
@@ -34,11 +34,23 @@ jobs:
     needs: [javadoc, typedoc, pydoc, cppdoc, rdoc]
     runs-on: ubuntu-22.04
     steps:
+      - name: Setup Git
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0      # Fixes Bug: https://github.com/actions/checkout/issues/1471
+          fetch-tags: true
+      - run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
This is a "hack" that just ensures we get a local tag as part of our `tag --sort` command. This should not be necessary; if we were to follow something like this, we should figure out an external sorting tool to use, and combine the output of `git tag` with our new tag, bypassing the need to do any sort of `git config`. That said, we need to figure out how we are going to solve this more generally (ie, outside the context of just docs CI). It _may_ very well be a solution that looks like this, but we need to figure it out more broadly.